sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -566,8 +568,36 @@ static struct snd_soc_dai_link *sof_card_dai_links_create(struct device *dev,
 			if (!links[id].cpus->dai_name)
 				goto devm_err;
 		}
+		id++;
 	}
 
+	links[id].name = "Compress Probe Playback";
+	links[id].id = id;
+	links[id].cpus = &cpus[id];
+	links[id].num_cpus = 1;
+	links[id].cpus->dai_name = "Probe Injection0 CPU DAI";
```

Comment:
What is Injection0? Could there be more?

Also, where are the CPU DAI's defined?

Reply:
CPU are defined at: sound/soc/sof/intel/hda-dai.c (~470).
Yeah, there can be many injections, although for current release, the expectation is: 1extraction probe point and 1x injection. Compress operations/ Probe IPCs are prepared to handle stress test scenarios with full bandwidth, however, I didn't put enough time yet in figuring out the correct way of initializing the exact N number of injectors given the hardware capabilities.

Reply:
@crojewsk which patch adds the CPU DAI's in this series?

Reply:
found it :d. sorry IM going blind reviewing the whole series

Reply:
@ranj063 No problem. I appreciate each and everyone taking his/her own time in reviewing the code. Whether blind or deaf, reviews almost always lead to a better quality code.

You see, when developing probes for SOF, I even found 1 bug and 1 improvement for cAVS - didn't catch them earlier because some IPCs never failed there, while with initial Arthur's FW release, most IPCs had problems.